samples/scripts/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Is this file needed?

Reply:
Definitely not, slipped in by mistake :-)